serviceUsage.html

```diff
@@ -69,7 +69,48 @@ <h2>
             So users should create a wrapper project to include both Generate/Publish and their protocol in
             or place the protocol library in the folder for external dependencies of their JVM installation.
         </p>
+        <h3>Jasypt configurations</h3>
+        <p>Jasypt Spring Boot provides Encryption support for property sources(passwords, secret info ..etc) in Spring Boot Applications. To support this functionality in our application we need to add the following property in property file</p>
```

Comment:
Please check on the line formats, it should be similar to break lines already existing.